src/PluginRegistrationDialog.cpp

```diff
@@ -430,452 +92,194 @@ void PluginRegistrationDialog::Populate()
 /// Defines the dialog and does data exchange with it.
 void PluginRegistrationDialog::PopulateOrExchange(ShuttleGui &S)
 {
-   S.StartVerticalLay(true);
+   constexpr int Margin = 12;
+
+   static const TranslatableStrings hiddenProviders = {
+      XO("Builtin Commands")
+   };
+   static const std::unordered_map<TranslatableString, TranslatableString> nameSubstitutions = {
+      { XO("Builtin Effects"), XO("Audacity Effects") }
```

Comment:
One small thing: Currently, sorting by "Generator", the "Type" menu saying "Audacity Effects"/"Nyquist Effects"/... is confusing, as you really are seeing something that's not an effect. Can you rename them to the following instead?

* Audio Unit 
* LADSPA 
* LV2 
* VST 
* VST3 
* Nyquist 
* Native

(where native is what currently is "Audacity Effects")